### DISCUSSION TITLE
Expand the time search range

### DISCUSSION THREAD

**Discussion by mikebyte2000** [MacielG1/VodRecovery#13] [Ideas]
If ✖  Unable to recover the video! then the program expanded the video search by -5 minutes and +5 minutes, because it is usually enough to specify a different time for the program to find a link to the video, and unfortunately it has to be done manually
For example, the program received the time 2024-07-07 20:20:00 from Twitchtracker/Streamscharts/Sullygnome, if the video could not be found, the range will increase from 20:15:00 to 20:25:00, skipping the time in which the program already searched for the video it is 20:20:00

**Comment by MacielG1** [comment]
There's no need to search within a time range. If the stream isn't found at the specific time provided, it won't be found at all. The program already handles variations in the exact second by checking from 00 to 59 seconds for the given time if any of the three websites (Twitchtracker, Streamscharts, or Sullygnome) don't return a specific second for the stream
